experimental/framework/tree-react-api/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Compiled TypeScript
```

Comment:
Is this file needed? I think it can just inherit properties from the root config to capture both of these.

Reply:
removed